assets/scss/pages/_page-article.scss

```
@@ -0,0 +1,21 @@
+@use '../abstracts' as *;
+
+.page-article {
+  .banner {
+    height: 1500px;
+    background-image: url('../../images/blog/pine_trees_background.jpg') !important;
```

C'est la même image que tu avais mis dans la bannière de l'index du blog ? Même remarque qu'à l'époque : elle est bcp trop grande et bcp trop lourde (presque 6 MB 😱). Supprime cette image dans le projet, et ici remplace la par celle qu'on avait créée qui est + petite et 6 fois moins lourde.

```suggestion
    background-image: url('../../images/blog/background.jpg') !important;
```

---

Indépendamment de ça, notre norme pour le nommage des images c'est les - et pas les _ (`pine_trees_background` -> `pine-trees-background`)